config-windows.md

```diff
@@ -40,15 +40,15 @@ The following parameters can be specified:
 
 * **`shares`** *(uint16, OPTIONAL)* - specifies the relative weight to other containers with CPU shares. The range is from 1 to 10000.
 
-* **`percent`** *(uint, OPTIONAL)* - specifies the percentage of available CPUs usable by the container.
+* **`maximum`** *(uint, OPTIONAL)* - specifies the portion of processor cycles that this container can use as a percentage times 100 . Range is from 1 to 10000.
```

Comment:
(Not a maintainer) but I concur with @darrenstahlmsft - on Windows, the ranges probably should remain in the OCI spec.